src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
```

Comment:
initialization in the constructor is better practice and I see that you did it in FigureSupplier, so better write your code more consistent

Reply:
done